scipy/optimize/_differentialevolution.py

```diff
@@ -309,8 +399,49 @@ def __init__(self, func, bounds, args=(),
                  strategy='best1bin', maxiter=None, popsize=15,
                  tol=0.01, mutation=(0.5, 1), recombination=0.7, seed=None,
                  maxfun=None, callback=None, disp=False, polish=True,
-                 init='latinhypercube'):
-
+                 init='latinhypercube', workers=None):
```

Comment:
The docstring says that `workers` should be a `pool` object or `int`. However, the default is `None`, which isn't one of the options given. Perhaps the default should be `workers=1`, in which case the code should use the serial case.